tests/test_ecephys/test_tools_spikeinterface.py

```diff
@@ -759,22 +776,6 @@ def test_row_matching_by_channel_name_with_new_property(self):
         self.assertListEqual(list(self.nwbfile.electrodes["channel_name"].data), expected_names)
         self.assertListEqual(list(self.nwbfile.electrodes["property"].data), expected_property_values)
 
-    def test_assertion_for_id_collision(self):
```

Comment:
Is there a test, either direct or indirect, that tests for this new non-collision for SI recordings?

Reply:
Yes, this test: https://github.com/catalystneuro/neuroconv/blob/multi-probe/tests/test_ecephys/test_tools_spikeinterface.py#L1149

It adds two recordings with the same channel ids, but different groups.

Reply:
That test is pretty heavy integration-wise, already a lot going on in it. I'd feel a lot better about this one if you restored the original structure (write two electrodes manually, then call `add_electrode` as a unit test), just adjust the ground truth to show the new behavior (to not raise error on collision)